lib/web_ui/lib/src/ui/text.dart

```diff
@@ -131,7 +131,7 @@ class FontWeight {
   ///
   /// Values for `t` are usually obtained from an [Animation<double>], such as
   /// an [AnimationController].
-  static FontWeight lerp(FontWeight a, FontWeight b, double t) {
+  static FontWeight/*?*/ lerp(FontWeight/*?*/ a, FontWeight/*?*/ b, double/*!*/ t) {
```

Comment:
`values` doesn't have any null entries:

```suggestion
  static FontWeight/*!*/ lerp(FontWeight/*?*/ a, FontWeight/*?*/ b, double/*!*/ t) {
```

Reply:
Right, but there's also:

```dart
    if (a == null && b == null)
      return null;
```

Reply:
Sorry, I don't follow =) Could you spell it out for me a bit more.

Reply:
Oh! I was looking at the `lib/ui/text.dart`, where the implementation is as follows:

```dart
  static FontWeight/*?*/ lerp(FontWeight/*?*/ a, FontWeight/*?*/ b, double/*!*/ t) {
    assert(t != null);
    if (a == null && b == null)
      return null;
    return values[lerpDouble(a?.index ?? normal.index, b?.index ?? normal.index, t).round().clamp(0, 8) as int];
  }
```

I should just sync the two so they are consistent. Both should be returning `FontWeight?` though.

Reply:
Done.